cmd/auth/in_memory_test.go

```diff
@@ -23,4 +24,4 @@ func (i *InMemoryTokenCache) Store(key string, t *oauth2.Token) error {
 	return nil
 }
 
-var _ TokenCache = (*InMemoryTokenCache)(nil)
+var _ cache.TokenCache = (*InMemoryTokenCache)(nil)
```

Comment:
Can we remove this line of code? What's the purpose?

Reply:
This line just ensures that the InMemoryTokenCache implements the cache.TokenCache interface, similar to OAuthEndpointSupplier and MockApiClient below.